app/src/main/java/nl/tudelft/cs4160/identitychain/main/bluetooth/BluetoothActivity.kt

```diff
@@ -33,7 +33,7 @@ class BluetoothActivity : AppCompatActivity(), CommunicationListener {
         Log.e(TAG, "pressed " + device.name + "\nUUID: " + device.uuids[0].uuid)
 
         val peer = Peer(device)
-        communication!!.connectToPeer(peer)
+        communication!!.connectToPeer(peer, "HELLO BLUETOOTH!") //FIXME either we dump bluetooth or we need to insert the payload here
```

Comment:
i say dump it, it's buggy at best anyways

Reply:
http://i.imgur.com/uFztWjz.gif